rmgpy/kinetics/arrhenius.pyx

```diff
@@ -249,7 +249,23 @@ cdef class Arrhenius(KineticsModel):
 
         # Set the rate parameter to a cantera Arrhenius object
         ctReaction.rate = self.toCanteraKinetics()
-    
+
+    cpdef ArrheniusEP toArrheniusEP(self):
```

Comment:
Sorry, I guess it's a little bit more complex than that.  Knowing Ea and alpha is sufficient to back out E0 which with alpha defines the EP kinetics.  

The reason I think we should have this is that I generally believe that functions should be general to all cases they might be used in and operate in a way that makes sense to the uninitiated.  This function sounds like it does something that physically make sense:  if you can estimate the alpha and you know the dHrxn and Ea you can generate an EP rule.  Instead it only makes rules with E0 = Ea, which makes no physical sense unless alpha really is 0.  If someone were looking to convert an Arrhenius object into a real EP rule, this function would look like what they were suppose to use, when in fact it does nothing related to that at all.  However, luckily in this case we can relatively easily make it function the physically sensical way if alpha is set and the programmatic way if alpha is left out, satisfying both possibilities.  

At least I think this is easy to do?